showcase/BUILD.bazel

```diff
@@ -52,6 +65,8 @@ java_gapic_assembly_gradle_pkg(
     transport = "grpc+rest",
     deps = [
         ":showcase_java_gapic",
+        # TODO(lawrenceqiu): Not adding :showcase_java_grpc_extended dep as that includes WickedGrpc.java
+        # Need to figure out why it's being included
```

Comment:
I'm not able to figure out why this is the case. Thanks @burkedavison for helping me with the rest of the bazel build rules 😄 !